pre-publish.sh

```diff
@@ -4,7 +4,7 @@
 wasm-pack build --target nodejs
 
 # Fix the name 
-sed -i -e 's/schnorrkel-js/@parity\/schnorrkel-js/g' pkg/package.json
+sed -i -e 's/schnorrkel-js/@substrate\/schnorrkel-js/g' pkg/package.json
```

Comment:
`wasm-pack` has the `--scope` flag, so you can do `wasm-pack build --target nodejs --scope substrate` and it will magically add the `@substrate/schnorrkel-js` part for you, one less `sed`

Reply:
Nice. This will remove the need for all of this manual bash-ing stuff. Will soon address this.